src/index.html

```diff
@@ -1,13 +1,72 @@
 <!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+<html lang="en" class="page">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./styles/style.css">
+  </head>
+  <body>
+
+    <div data-qa="card" class="card card--container">
+      <div class="card__photo-container">
+        <img
+          class="card__photo"
+          src="images/imac.jpeg"
+          alt="imac"
+        >
+      </div>
+
+      <div class="card__text">
+        <h2 class="card__desc">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+      </div>
+  
+      <div class="card__code">
+        <p class="card__product-code">
+          Product code: 195434
+        </p>
+      </div>
+  
+      <div class="stars">
+        <div class="stars__card">
+          <ul class="stars__list">
+            <li class="stars__item stars__item--one"></li>
+            <li class="stars__item stars__item--two"></li>
+            <li class="stars__item stars__item--three"></li>
+            <li class="stars__item stars__item--four"></li>
+            <li class="stars__item stars__item--five"></li>
+          </ul>
```

Comment:
This task isn't implemented. I can't change the amount of highlighted stars now 
<img width="906" alt="image" src="https://user-images.githubusercontent.com/99403198/210062483-e39bbcef-04b2-4e6b-ae66-c2d68f1f7e03.png">